staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/plugin.go

```diff
@@ -127,7 +127,7 @@ func (a *Plugin) Validate(ctx context.Context, attr admission.Attributes, o admi
 }
 
 func (a *Plugin) InspectFeatureGates(featureGates featuregate.FeatureGate) {
-	a.Plugin.SetEnabled(featureGates.Enabled(features.ValidatingAdmissionPolicy))
+	a.Plugin.SetEnabled(true)
```

Comment:
is there anything initializing these plugins with enabled=false any more? should we drop that capability from the generic class and assume it is always enabled now?

Reply:
This capability from generic class would also be used for MutatingAdmissionPolicy hence we would like to leave it there as of now...

Reply:
ok

Reply:
rather than doing this in InspectFeatureGates, seems like we can just hard-code true in NewPlugin?